frontend/src/components/MeetingRoom/PopupDialog/PopupDialog.tsx

```diff
@@ -51,7 +51,7 @@ const PopupDialog = ({
       </DialogContent>
       <DialogActions>
         <Button onClick={handleClose}>{actionText.secondaryActionText}</Button>
-        <Button onClick={handleActionClick} autoFocus data-testId="popup-dialog-primary-button">
+        <Button onClick={handleActionClick} autoFocus data-testid="popup-dialog-primary-button">
```

Comment:
An annoying console warning, pet project I'll handle as soon as I'm done with the camera orientation PR

Reply:
finally someone removed it 😅